tools/lint.toml

```diff
@@ -20,8 +20,7 @@ target-version = "py39"
 "scipy/stats/distributions.py" = ["F403", "F405"]
 "scipy/stats/mstats.py" = ["F403", "F401"]
 "scipy/stats/qmc.py" = ["F403"]
-"scipy/optimize/_linprog.py" = ["F401"]
-"scipy/optimize/_linprog_ip.py" = ["F401"]
+"scipy/stats/tests/data/_mvt.py" = ["E701", "E702"]
```

Comment:
I assume the multiple statements on one line are intentional here, @mdhaber ?

Reply:
Yes. The intent was to translate a Matlab function as directly as possible.